xml/System.Windows.Forms/ComboBox+ObjectCollection.xml

```diff
@@ -1,867 +1,876 @@
+      <MemberSignature Language="ILAsm" Value=".method hidebysig newslot virtual instance int32 System.Collections.Generic.IComparer&lt;System.Windows.Forms.ComboBox.ObjectCollection.Entry&gt;.Compare(class System.Windows.Forms.ComboBox/ObjectCollection/Entry entry1, class System.Windows.Forms.ComboBox/ObjectCollection/Entry entry2) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.Windows.Forms.ComboBox.ObjectCollection.System#Collections#Generic#IComparer&lt;System#Windows#Forms#ComboBox#ObjectCollection#Entry&gt;#Compare(System.Windows.Forms.ComboBox.ObjectCollection.Entry,System.Windows.Forms.ComboBox.ObjectCollection.Entry)" />
+      <MemberSignature Language="VB.NET" Value="Function Compare (entry1 As ComboBox.ObjectCollection.Entry, entry2 As ComboBox.ObjectCollection.Entry) As Integer Implements IComparer(Of ComboBox.ObjectCollection.Entry).Compare" />
+      <MemberSignature Language="F#" Value="abstract member System.Collections.Generic.IComparer&lt;System.Windows.Forms.ComboBox.ObjectCollection.Entry&gt;.Compare : System.Windows.Forms.ComboBox.ObjectCollection.Entry * System.Windows.Forms.ComboBox.ObjectCollection.Entry -&gt; int&#xA;override this.System.Collections.Generic.IComparer&lt;System.Windows.Forms.ComboBox.ObjectCollection.Entry&gt;.Compare : System.Windows.Forms.ComboBox.ObjectCollection.Entry * System.Windows.Forms.ComboBox.ObjectCollection.Entry -&gt; int" Usage="objectCollection.System.Collections.Generic.IComparer&lt;System.Windows.Forms.ComboBox.ObjectCollection.Entry&gt;.Compare (entry1, entry2)" />
+      <MemberSignature Language="C++ CLI" Value=" virtual int System.Collections.Generic.IComparer&lt;System.Windows.Forms.ComboBox.ObjectCollection.Entry&gt;.Compare(System::Windows::Forms::ComboBox::ObjectCollection::Entry ^ entry1, System::Windows::Forms::ComboBox::ObjectCollection::Entry ^ entry2) = System::Collections::Generic::IComparer&lt;System::Windows::Forms::ComboBox::ObjectCollection::Entry ^&gt;::Compare;" />
+      <MemberType>Method</MemberType>
+      <Implements>
+        <InterfaceMember>M:System.Collections.Generic.IComparer`1.Compare(`0,`0)</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Int32</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="entry1" Type="System.Windows.Forms.ComboBox+ObjectCollection+Entry" Index="0" FrameworkAlternate="net-6.0" />
+        <Parameter Name="entry2" Type="System.Windows.Forms.ComboBox+ObjectCollection+Entry" Index="1" FrameworkAlternate="net-6.0" />
+      </Parameters>
+      <Docs>
+        <param name="entry1">To be added.</param>
+        <param name="entry2">To be added.</param>

```

Comment:
These parameters were not documented in triple slash comments. Can you please manually add documentation for them?